README.md

```diff
@@ -51,12 +136,24 @@ pip install pytest
 pytest test
 ```
 
-TODO
------
+Migration Guide
+----------------
 
-Get the docs on read the docs
+The `setgapth.py` way to install is now deprecated by `pip install galgebra` and all modules in GAlgebra should be imported from `galgebra`, for example:
 
-Ensure functions are documented
+```python
+from galgebra.printer import Format, Eprint, Get_Program, latex, GaPrinter
+from galgebra.ga import Ga, one, zero
+from galgebra.mv import Mv, Nga
+# for backward compatibility
+from galgebra.mv import MV, ONE, ZERO, HALF
+from galgebra import ga
+from galgebra import utils
+```
 
-Set up proper release version system
+Resources
+------------
 
+Note that in the doc directory there is BookGA.pdf which is a collection of notes on 
+Geometric Algebra and Calculus based of "Geometric Algebra for Physicists" by Doran and 
+Lasenby and on some papers by Lasenby and Hestenes.
```

Comment:
I'm not sure I understand what "and on" means here. Also (possibly related), the "based of" in the line above should probably be "based on" or "based off".

Reply:
I don't understand what these line mean exactly too. I thought these lines are for attribution to the authors of the materials included in this repo, there're probably some concerns about copyright and consent, hence the "Note".